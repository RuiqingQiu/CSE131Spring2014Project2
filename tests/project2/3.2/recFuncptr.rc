function : int foo(int x){
   static auto this = foo;
   if (x > 0){
      cout << x;
      return 1 + this(--x);
   }

   return 0;
}
function : int main(){
   cout << foo(15) << endl;
   return 0;
}
