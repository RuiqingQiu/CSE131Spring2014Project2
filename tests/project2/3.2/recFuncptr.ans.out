15141312111098765432115
